src/app/components/tables/simple/terra-simple-table.component.html

```diff
@@ -23,14 +23,14 @@
         </tr>
         </thead>
         <tbody [class.highlighting]="inputUseHighlighting">
-        <ng-template ngFor let-row [ngForOf]="inputRowList">
+        <ng-template ngFor let-row [ngForOf]="_rowList">
```

Comment:
When you use ng-container, you can use the normal *ngFor syntax. Maybe you can refactor the whole template using ng-container 😃 

Use rowList getter